.github/workflows/release-fhir-server.yml

```
@@ -0,0 +1,92 @@
+
+env:
+  BALLERINA_VERSION: 2201.12.10
+  WORKING_DIR: miscellaneous/fhir-server
+
+jobs:
+  build-and-zip:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)